src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/LessAndGreaterThanCompletionSession.cs

```diff
@@ -58,8 +59,19 @@ private bool PossibleTypeArgument(ITextSnapshot snapshot, SyntaxToken token, Can
             }
 
             var model = document.GetSemanticModelAsync(cancellationToken).WaitAndGetResult(cancellationToken);
-            var info = model.GetSymbolInfo(node.Left, cancellationToken);
 
+            // Analyze node on the left of < operator to verify if it is a generic type or method.
+            var leftNode = node.Left;
+            if (leftNode is ConditionalAccessExpressionSyntax)
+            {
+                // If node on the left is a conditional access expression, get the member binding expression 
+                // from the innermost conditional acccess expression, which is the left of < operator. 
+                // e.g: In a?.b?.c< we need to get the conditional access expression .b?.c and analyze its
+                // member binding expression (the .c) to see if it is a generic type/method.
+                leftNode = leftNode.GetInnerMostConditionalAccessExpression().WhenNotNull;
```

Comment:
No. that works. In that example, `GetInnerMostConditionalAccessExpression` returns `.b?.c.d` and its `WhenNotNull` member is a `SimpleMemberAccessExpression` - `.c.d` .

So long as I do not give a `ConditionalAccessExpression` (instead of the expression to right of ?.) to `GetSymbolInfo` we'll be fine.